src/Umbraco.Web.UI.Client/src/common/directives/components/forms/umbcheckbox.directive.js

```diff
@@ -50,7 +50,7 @@
 
         function onInit() {
             vm.inputId = vm.inputId || "umb-check_" + String.CreateGuid();
-
+            vm.disableDirtyCheck = vm.disableDirtyCheck === true;
```

Comment:
Is this necessary? And won't it break backwards compat with people using a truthly value that's not strictly `true`?